### PR TITLE
Added a counter for etw loops and slowed down retry rate.

### PR DESCRIPTION
**Pull request by bmcder02**
Added a fail counter for watch_etw and slowed the retry rate in reference to #3206 .  If this fails 10 times, it will now exit as there is likely a wider problem that's causing the error loop.

### CODE REVIEW

File: vql/windows/etw/watch_etw.go [hunk 1]
```diff
@@ -78,16 +78,17 @@ func (self WatchETWPlugin) Call(
 			EnableMapInfo: arg.EnableMapInfo,
 		}
 
-		for {
+		for i := 0; i < 10; i++ {
 			err = self.WatchOnce(ctx, scope, arg.Stop, output_chan,
```
Comment (scudette) [inline vql/windows/etw/watch_etw.go:82]: This needs to be an exponential back-off strategy without giving up.

Comment (bmcder02) [reply vql/windows/etw/watch_etw.go:83]: Changed it to a timer that doubles its backoff after each attempt.

File: vql/windows/etw/watch_etw.go [hunk 2]
```diff
@@ -78,12 +78,14 @@ func (self WatchETWPlugin) Call(
...
 		for {
 			err = self.WatchOnce(ctx, scope, arg.Stop, output_chan,
 				arg.Name, options, wGuid)
 			if err != nil {
-				scope.Log("watch_etw: ETW session interrupted, will retry again.")
-				utils.SleepWithCtx(ctx, 10*time.Second)
+				scope.Log("watch_etw: ETW session interrupted, will retry again in %d seconds: %v", timer, err)
+				utils.SleepWithCtx(ctx, timer)
+				timer *= 2
 				continue
```
Comment (scudette) [inline vql/windows/etw/watch_etw.go:89]: If the ETW works then we need to reset the timer

File: vql/windows/etw/watch_etw.go [hunk 3]
```diff
@@ -78,12 +78,14 @@ func (self WatchETWPlugin) Call(
 			EnableMapInfo: arg.EnableMapInfo,
 		}
 
+		timer := 10 * time.Second
```
Comment (scudette) [inline vql/windows/etw/watch_etw.go:81]: It should start off much faster than this - maybe a sec?

**Pull request merged**